tests/_support/Controllers/Popcorn.php

```diff
@@ -55,4 +55,19 @@ public function index3()
 		return $response;
 	}
 
+	public function canyon()
+	{
+		echo 'Hello-o-o';
+	}
+
+	public function json()
+	{
+		$this->responsd(['answer' => 42]);
```

Comment:
Typo in `response`. Not sure if intentional for this controller...

Reply:
Good catch, though it doesn't affect the tests - still messed :(